core/src/lib/var.cc

```diff
@@ -2482,79 +2482,80 @@ var_rc_t var_format(var_t* var,
 static const char* var_errors[] = {
-    _("everything ok"),                /* VAR_OK = 0 */
-    _("incomplete named character"),   /* VAR_ERR_INCOMPLETE_NAMED_CHARACTER */
-    _("incomplete hexadecimal value"), /* VAR_ERR_INCOMPLETE_HEX */
-    _("invalid hexadecimal value"),    /* VAR_ERR_INVALID_HEX */
-    _("octal value too large"),        /* VAR_ERR_OCTAL_TOO_LARGE */
-    _("invalid octal value"),          /* VAR_ERR_INVALID_OCTAL */
-    _("incomplete octal value"),       /* VAR_ERR_INCOMPLETE_OCTAL */
-    _("incomplete grouped hexadecimal value"), /* VAR_ERR_INCOMPLETE_GROUPED_HEX
-                                                */
-    _("incorrect character class specification"), /* VAR_ERR_INCORRECT_CLASS_SPEC
-                                                   */
-    _("invalid expansion configuration"),   /* VAR_ERR_INVALID_CONFIGURATION */
-    _("out of memory"),                     /* VAR_ERR_OUT_OF_MEMORY */
-    _("incomplete variable specification"), /* VAR_ERR_INCOMPLETE_VARIABLE_SPEC
-                                             */
-    _("undefined variable"),                /* VAR_ERR_UNDEFINED_VARIABLE */
-    _("input is neither text nor variable"), /* VAR_ERR_INPUT_ISNT_TEXT_NOR_VARIABLE
-                                              */
-    _("unknown command character in variable"), /* VAR_ERR_UNKNOWN_COMMAND_CHAR
-                                                 */
-    _("malformatted search and replace operation"), /* VAR_ERR_MALFORMATTED_REPLACE
-                                                     */
-    _("unknown flag in search and replace operation"), /* VAR_ERR_UNKNOWN_REPLACE_FLAG
-                                                        */
-    _("invalid regex in search and replace operation"), /* VAR_ERR_INVALID_REGEX_IN_REPLACE
-                                                         */
-    _("missing parameter in command"), /* VAR_ERR_MISSING_PARAMETER_IN_COMMAND
-                                        */
-    _("empty search string in search and replace operation"), /* VAR_ERR_EMPTY_SEARCH_STRING
-                                                               */
-    _("start offset missing in cut operation"), /* VAR_ERR_MISSING_START_OFFSET
-                                                 */
-    _("offsets in cut operation delimited by unknown character"), /* VAR_ERR_INVALID_OFFSET_DELIMITER
-                                                                   */
-    _("range out of bounds in cut operation"), /* VAR_ERR_RANGE_OUT_OF_BOUNDS */
-    _("offset out of bounds in cut operation"), /* VAR_ERR_OFFSET_OUT_OF_BOUNDS
+    T_("everything ok"),                /* VAR_OK = 0 */
+    T_("incomplete named character"),   /* VAR_ERR_INCOMPLETE_NAMED_CHARACTER */
+    T_("incomplete hexadecimal value"), /* VAR_ERR_INCOMPLETE_HEX */
+    T_("invalid hexadecimal value"),    /* VAR_ERR_INVALID_HEX */
+    T_("octal value too large"),        /* VAR_ERR_OCTAL_TOO_LARGE */
```

Comment:
For arrays such as this I much prefer the C syntax:
```suggestion
    [VAR_OK] = T_("everything ok"),
    [VAR_ERR_INCOMPLETE_NAMED_CHARACTER] = T_("incomplete named character"),
    [VAR_ERR_INCOMPLETE_HEX] = T_("incomplete hexadecimal value"),
    [VAR_ERR_INVALID_HEX] = T_("invalid hexadecimal value"),
    [VAR_ERR_OCTAL_TOO_LARGE] = T_("octal value too large"),
```
even if not c++ standard compliant.  But this change does not belong in this pr.